lib/paper_trail/compatibility.rb

```diff
@@ -18,7 +18,7 @@ module PaperTrail
   # versions.
   module Compatibility
     ACTIVERECORD_GTE = ">= 6.1" # enforced in gemspec
-    ACTIVERECORD_LT = "< 7.2" # not enforced in gemspec
+    ACTIVERECORD_LT = "< 7.3" # not enforced in gemspec
```

Comment:
8.0 ? 
```ruby
ACTIVERECORD_LT = "< 8.0"
```

Reply:
It was set to `< 8` originally but there was a request to set to `< 7.3` [here](https://github.com/paper-trail-gem/paper_trail/pull/1485#discussion_r1718292221).